Author: <jblache@debian.org>
Description: Remove check for SANE_CAP_ALWAYS_SETTABLE; the flag is not part
of the standard and has been removed upstream in SANE 1.0.20.
Index: trunk/src/gtkglue.c
===================================================================
--- trunk.orig/src/gtkglue.c
+++ trunk/src/gtkglue.c
@@ -1476,8 +1476,7 @@ gsg_set_sensitivity (GSGDialog * dialog,
 	  || opt->type == SANE_TYPE_GROUP || !dialog->element[i].widget)
 	continue;
 
-      if (!(opt->cap & SANE_CAP_ALWAYS_SETTABLE))
-	gtk_widget_set_sensitive (dialog->element[i].widget, sensitive);
+      gtk_widget_set_sensitive (dialog->element[i].widget, sensitive);
     }
 }
 
